kodilla-testing/src/main/java/com/kodilla/testing/shape/Square.java

```diff
@@ -0,0 +1,20 @@
+package com.kodilla.testing.shape;
+
+class Square implements Shape {
+
+    private final String shapeName;
+    private int fieldSize;
+
+    public Square(String shapeName, int fieldSize) {
+        this.shapeName = shapeName;
+        this.fieldSize = fieldSize;
```

Comment:
That should work a bit different. Square should have a constant field `private static final String NAME` with hardcoded value. The constructor should have a single input - e.g. sideSize. Internally you need to calculate the fieldSize based on the sideSize. The same applies to other shapes